grpc-gcp/src/main/java/com/google/grpc/gcp/GcpMetricsConstants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.grpc.gcp;
+
+public class GcpMetricsConstants {
```

Comment:
this only class I see this class being used is grpc-gcp/src/main/java/com/google/grpc/gcp/GcpManagedChannel.java , which is in the same package.
so can this class be package-private?

Reply:
I thought it will be needed in the tests but the package is the same, so it's not. Done.